code/game/gamemodes/malfunction/Malf_Modules.dm

```diff
@@ -40,10 +40,8 @@
 	doomsday_device = DOOM
 	doomsday_device.start()
 	verbs -= /mob/living/silicon/ai/proc/nuke_station
-	for(var/obj/item/weapon/pinpointer/point in pinpointer_list)
-		for(var/mob/living/silicon/ai/A in ai_list)
-			if((A.stat != DEAD) && A.nuking)
-				point.the_disk = A //The pinpointer now tracks the AI core
+	for(var/obj/item/weapon/pinpointer/P in pinpointer_list)
```

Comment:
Why, though? The only things in the list are pinpointers. I don't know how to optimize fors/lists easily so let me know if I need to change it.